.ci/jenkins/Jenkinsfile.buildchain

```diff
@@ -54,6 +54,21 @@ pipeline {
                 }
             }
         }
+        stage('Install build-chain tool') {
+            steps {
+                script {
+                    println "[INFO] Recovering build-chain version from composite action file"
```

Comment:
Recovering?

Reply:
probably Getting is a more appropriated word. Changed. Thanks!